src/Microsoft.ML.Data/Transforms/TermTransform.cs

```diff
@@ -196,6 +202,26 @@ private CodecFactory CodecFactory
         public override bool CanSavePfa => true;
         public override bool CanSaveOnnx => true;
 
+        /// <summary>
+        /// Convenience constructor for public facing API.
+        /// </summary>
+        /// <param name="env">Host Environment.</param>
+        /// <param name="input">Input <see cref="IDataView"/>. This is the output from previous transform or loader.</param>
+        /// <param name="name">Name of the output column.</param>
+        /// <param name="source">Name of the column to be transformed. If this is null '<paramref name="name"/>' will be used.</param>
+        /// <param name="maxNumTerms">Maximum number of terms to keep per column when auto-training.</param>
+        /// <param name="sort">How items should be ordered when vectorized. By default, they will be in the order encountered.
+        /// If by value items are sorted according to their default comparison, e.g., text sorting will be case sensitive (e.g., 'A' then 'Z' then 'a').</param>
+        public TermTransform(IHostEnvironment env,
+            IDataView input,
+            string name,
+            string source = null,
+            int maxNumTerms = Defaults.MaxNumTerms,
+            SortOrder sort = Defaults.Sort)
+            : this(env, new Arguments() { Column = new[] { new Column() { Name = name, Source = source ?? name } }, MaxNumTerms = maxNumTerms, Sort = sort }, input)
```

Comment:
>Name = name, Source = source ?? name  [](start = 74, length = 37)

just nitpicking, everywhere else you use Source = source?? name, Name = name, and here you have different order!